### PR TITLE
Agl/add is language of consultiung boolean

### PR DESCRIPTION
**Pull request by adam-soltech**
[ticket](https://seed-company-squad.monday.com/boards/3451697530/views/78801638/pulses/4788464835?filter=XQAAAALKAAAAAAAAAABBqQqHk62Wwa8yt9cbbXI48HpL2ei1XixEiNVOhiWmf4IrTn4lrcVxz0d8EzlqDpxL1dc7i9JjI1nNW0HEFgqaXhsDsHWTHjMEE6-qlUKWgWTaunFWUdBXUlo9GrL7-Dxs3yXtVsCpk7bU6tq4ASH03XfQsurYW6oz6SfWXIHjSrPNZIVE3__87GiYdcf5oUuHV6IpVDC18IDUOyTd3hSlyn4xwV6TGJGJ____cnOAAA)

┆Issue is synchronized with this [Monday item](https://seed-company-squad.monday.com/boards/3451697530/pulses/5640489530) by [Unito](https://www.unito.io)

### CODE REVIEW

File: dbschema/migrations/00033.edgeql
Comment (bryanjnelson) [inline dbschema/migrations/00033.edgeql:1]: You wouldn't have known this, but we're not actually checking in EdgeDB Migrations at this point.  Carson is handling this himself, and grouping them together as makes sense.  Eventually that will change after we're fully migrated, but for now don't worry about doing the migration.

**Review by bryanjnelson — CHANGES_REQUESTED** [review]
Thanks @adam-soltech!

**Comment by CarsonF** [comment]
`Language.isLanguageOfConsulting` seems not the best name since it has the object name in it again.
- `isUsedForConsulting`?
- `canBeUsedForConsulting`?
- `allowedForConsulting`?
- `consulting`?

maybe an enum list would be better to encompass all of these?
```ts
Language.allowedUse = ['WiderCommunication', 'Consulting', 'X'];
```
Though I guess it would be better to describe what this language _is_ vs how the language is allowed to be used in other places.
If we wanted different roles/conditions to be able to assign individual values, then a single list isn't the best way.

File: dbschema/migrations/00033.edgeql
Comment (adam-soltech) [reply dbschema/migrations/00033.edgeql:1]: Without the migration the [e2e tests failed](https://github.com/SeedCompany/cord-api-v3/actions/runs/7134600634/job/19429741481)

Comment (bryanjnelson) [reply dbschema/migrations/00033.edgeql:1]: Ah yes.  Then for now just eliminate the EdgeDB additions completely.  We'll attack that in a separate PR altogether.

Comment (adam-soltech) [reply dbschema/migrations/00033.edgeql:1]: That also breaks the build. I sent the commit without edgedb and broken build to the dev channel

**Comment by CarsonF** [comment]
`isUsed` seems like the least offensive prefix. I wonder if `isUsedWithConsultants` would be better than `ForConsulting`?

**Comment by bryanjnelson** [comment]
> `isUsed` seems like the least offensive prefix. I wonder if `isUsedWithConsultants` would be better than `ForConsulting`?

If the UI is going to say "for consulting" then I'd just as soon match that.  Not much is gained otherwise imo.

**Comment by CarsonF** [comment]
I guess that's a question for product.

**Comment by adam-soltech** [comment]
> > `isUsed` seems like the least offensive prefix. I wonder if `isUsedWithConsultants` would be better than `ForConsulting`?
> 
> If the UI is going to say "for consulting" then I'd just as soon match that. Not much is gained otherwise imo.

I agree but will run it by product in standup tomorrow

File: dbschema/migrations/00033.edgeql
Comment (bryanjnelson) [reply dbschema/migrations/00033.edgeql:1]: You'll still want to take both the .edgql and .esdl files out of this PR.  Edge stuff is being done separately.

**Comment by sethmcknight** [comment]
This lived to the end of its lifetime and died

**Pull request closed without merging**